pkg/importer/imageio-datasource.go

```diff
@@ -52,23 +52,29 @@ type ImageioDataSource struct {
 	imageTransfer *ovirtsdk4.ImageTransfer
 	// connection is connection to the oVirt system
 	connection ConnectionInterface
+	// currentSnapshot is the UUID of the snapshot to copy, if requested
+	currentSnapshot string
+	// previousSnapshot is the UUID of the parent snapshot, if requested
+	previousSnapshot string
 }
 
 // NewImageioDataSource creates a new instance of the ovirt-imageio data provider.
-func NewImageioDataSource(endpoint string, accessKey string, secKey string, certDir string, diskID string) (*ImageioDataSource, error) {
+func NewImageioDataSource(endpoint string, accessKey string, secKey string, certDir string, diskID string, currentCheckpoint string, previousCheckpoint string, finalCheckpoint string) (*ImageioDataSource, error) {
```

Comment:
is finalCheckpoint used anywhere?

Reply:
I thought it would be when I started, but the ImageIO data source itself ended up not needing it. I took this out.